CHANGELOG.md

```diff
@@ -5,13 +5,22 @@
 **Breaking Changes**:
 
 - The `sentry_options_set_logger` function now accepts a `userdata` parameter.
-- The `name` parameter of `sentry_options_add_attachment(w)` was removed, it will
-  now be inferred from the filename of `path`.
+- The `name` parameter of `sentry_options_add_attachment(w)` was removed, it will now be inferred from the filename of `path`.
 
 **Features**:
 
+- Release Health (Sessions) is now stable, and automatic session handling is enabled by default. Use `sentry_options_set_auto_session_tracking` to change the behavior.
```

Comment:
This is a major new feature, and we could phrase this even more user-oriented and link to our Release Health docs at https://docs.sentry.io/workflow/releases/health/. How about:

```suggestion
- [Release Health](https://docs.sentry.io/workflow/releases/health/) support is now stable and enabled by default. After the update, you will see the number of crash free sessions and crash free users on the Releases page in Sentry. To customize sessions, use: <code block underneath> To disable automatic session tracking, use `sentry_options_set_auto_session_tracking`.
```